src/utilities/validateReport.js

```diff
@@ -0,0 +1,87 @@
+/**
+ * Check if the given parameter is a valid Report based
+ * on its data structure and type and return a boolean.
+ *
+ * @param   {object}  parsedData  The data to check.
+ * @param   {string}  type        Type of Report to check.
+ */
+
+import Ajv from 'ajv';
+
+// Expected json structure for Table reports
+const tableReportSchema = {
+  type: 'object',
+  properties: {
+    data: {
+      type: 'object',
+      properties: {
+        header: {
+          type: 'array',
```

Comment:
does this get an `items`?

Reply:
Yeah, I missed that. Sure, this line should be added: `items: { type: 'string' }`
Thanks!

Reply:
Done here e7779b1